internal/healthchecks/network_check.go

```diff
@@ -20,9 +20,38 @@ import (
 )
 
 var (
-	// API urls
-	loggingAPIUrl    = "https://logging.googleapis.com/$discovery/rest"
-	monitoringAPIUrl = "https://monitoring.googleapis.com/$discovery/rest"
+	requests = [...]networkCheckRequest{
```

Comment:
Optional: since `requests` is only used once, you could define both the struct and its elements inline within `RunCheck` and avoid needing to define a named struct. (This is a common pattern used in unit tests too: https://gobyexample.com/testing)